packages/runtime/src/runtimes/react/testing/react-provider.tsx

```diff
@@ -0,0 +1,19 @@
+import { RootStyleRegistry } from '../../../next'
+import { ReactRuntimeProvider } from '../components/RuntimeProvider'
+import { ReactRuntime } from '../react-runtime'
+
+export function ReactProvider({
+  children,
+  runtime,
+  previewMode = false,
+}: {
```

Comment:
An equivalent of the userland [`MakeswiftProvider`](https://docs.makeswift.com/developer/app-router/installation) for rendering tests.